notes/fidelity.html

```diff
@@ -71,6 +66,11 @@ <h2 class="ui header">
         <li>
             Call Fidelity Account Key Support at +1 800-544-6666.
         </li>
+        <li>
```

Comment:
The organization of this list is not correct. As documented at https://www.fidelity.com/security/extra-security-login, Fidelity has two separate routes to two different kinds of 2FA for logins. You can either set up the Symantec VIP Access proprietary 2FA on logins as before (using either a physical token or the proprietary apps) which still requires a phone call, OR you can set up 2FA via a phone number (SMS or automated voice call) which should be set up through the online security center.

It's not obvious to me whether it's better to try to write your own documentation of this here, or to simply link to (or quote directly from?) the page at Fidelity.com which seems to cover it pretty well: https://www.fidelity.com/security/extra-security-login